.github/codecov.yml

```diff
@@ -0,0 +1,11 @@
+coverage:
+  status:
+    project:
+      default:
+        target: 75
+        threshold: .5
+    patch:
+      default:
+        target: 75
+        threshold: .5
```

Comment:
should this file be in gitignore?

Reply:
After today, it should not be changed...but I had to add it this branch to get it to pass coverage because of the changes to testing.